maintenance/wikia/HttpsMigration/migrateWikiToFandom.php

```diff
@@ -101,13 +114,17 @@ public function execute() {
 			}
 
 			$this->output( "Wiki with ID {$sourceWikiId} was migrated from {$sourceDomain} to {$targetDomain}!\n" );
+			if ( strpos($targetDomain, "/") === false ){
```

Comment:
```suggestion
			if ( !empty( $additionalTargetDomain )  ){
```
or just move it below the actual action.